app/[locale]/what-is-ethereum-network/page.tsx

```diff
@@ -0,0 +1,761 @@
+}
+
+const Page = async ({ params }: { params: Promise<{ locale: Lang }> }) => {
+  const { locale } = await params
+
+  setRequestLocale(locale)
+
+  const t = await getTranslations({
+    locale,
+    namespace: "page-what-is-ethereum-network",
+  })
+
+  const commitHistoryCache: CommitHistory = {}
+  const { contributors, lastEditLocaleTimestamp } =
+    await getAppPageContributorInfo(
+      "what-is-ethereum-network",
+      locale as Lang,
+      commitHistoryCache
+    )
+
+  const heroProps: ContentHeroProps = {
+    breadcrumbs: { slug: "learn/what-is-ethereum-network", startDepth: 1 },
+    heroImg,
+    title: t("page-what-is-ethereum-network-title"),
+    description: (
+      <>
+        <p>{t("page-what-is-ethereum-network-description-1")}</p>
+        <p>{t("page-what-is-ethereum-network-description-2")}</p>
+      </>
+    ),
+  }
+
+  const tocItems: ToCItem[] = [
+    {
+      title: t("page-what-is-ethereum-network-title"),
+      url: "#ethereum-network",
+    },
+    {
+      title: t("page-what-is-ethereum-network-section-network-fees-title"),
+      url: "#network-fees",
+    },
+    {
+      title: t("page-what-is-ethereum-network-section-staking-title"),
+      url: "#staking",
+    },
+    {
+      title: t("page-what-is-ethereum-network-section-layer-2s-title"),
+      url: "#layer-2s",
+    },
+    {
+      title: t("page-what-is-ethereum-network-section-live-network-data-title"),
+      url: "#live-network-data",
+    },
+  ]
+
+  const getId = (input: string) => {
+    const parts = input.split("#")
+    return parts.length > 1 ? parts[1] : ""
+  }
+
+  return (
+    <>
+      <ContentHero {...heroProps} />
+      <MainArticle className="grid w-full grid-cols-1 gap-x-20 px-4 py-8 lg:grid-cols-[1fr_auto] lg:px-10 lg:py-10">
+        <div
+          data-label="extras"
+          className="mb-8 space-y-4 lg:col-start-1 lg:row-start-1 lg:mb-10"
+        >
+          <FileContributors
+            className="!py-0 [&>div]:mt-0"
+            contributors={contributors}
+            lastEditLocaleTimestamp={lastEditLocaleTimestamp}
+          />
+        </div>
+
+        <div className="row-start-1 lg:col-start-2 lg:row-span-2">
+          <TableOfContents variant="beginner" items={tocItems} isMobile />
+          <TableOfContents variant="beginner" items={tocItems} />
+        </div>
+
+        <div className="max-w-[50rem] space-y-14 lg:col-start-1 lg:row-start-2">
+          <Section id={getId(tocItems[0].url)} className="space-y-6">
+            <p>{t("page-what-is-ethereum-network-section-description-1")}</p>
+            <p>
+              {t.rich("page-what-is-ethereum-network-section-description-2", {
+                strong: Strong,
+              })}
+            </p>
+            <p>
+              {t.rich("page-what-is-ethereum-network-section-description-3", {
+                strong: Strong,
+              })}
+            </p>
+            <Image src={computerImg} alt="Computer" className="mx-auto" />
```

Comment:
Noted below, but need `sizes` on this... this one should be as simple as just 301px

```suggestion
            <Image
              src={computerImg}
              alt="Computer"
              className="mx-auto"
              sizes="301px"
            />
```